src/jllvm/vm/NativeImplementation.hpp

```diff
@@ -146,7 +161,53 @@ class ClassModel : public ModelBase<ClassObject>
     constexpr static llvm::StringLiteral className = "java/lang/Class";
     constexpr static auto methods =
         std::make_tuple(addMember<&ClassModel::registerNatives>(), addMember<&ClassModel::isArray>(),
-                        addMember<&ClassModel::desiredAssertionStatus0>());
+                        addMember<&ClassModel::desiredAssertionStatus0>(), addMember<&ClassModel::getPrimitiveClass>());
+};
+
+class FloatModel : public ModelBase<>
+{
+public:
+    using Base::Base;
+
+    static std::uint32_t floatToRawIntBits(VirtualMachine&, GCRootRef<ClassObject>, float value)
+    {
+        std::uint32_t repr;
+        std::memcpy(&repr, &value, sizeof(float));
+        return repr;
```

Comment:
```suggestion
        return std::bit_cast<std::uint32_t>(value);
```

Reply:
`std::bit_cast` is only supported since GCC 11 and Clang 14. Not sure which clang version the apple clang version on the Mac runner maps to but our Ubuntu runner is still using GCC 10.
We can definitely discuss upgrading it. I'll just leave a TODO here for now instead.